model/src/w3fldsmd.F90

```diff
@@ -25,10 +25,10 @@ MODULE W3FLDSMD
 !/    06-Dec-2010 : Change from GLOBAL (logical) to ICLO (integer) to
 !/                  specify index closure for a grid.   ( version 3.14 )
 !/                  (T. J. Campbell, NRL)
-!/    30-Oct-2012 : Implement tidal analysis            ( version 4.08 ) 
+!/    30-Oct-2012 : Implement tidal analysis            ( version 4.08 )
```

Comment:
This entire file is just white space changes.  Should this file be reverted to it's original?

Reply:
Again, this is stripping of trailing whitespace.